packages/yarnpkg-cli/sources/boot-cli-dev.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+require('../../../.pnp.js').setup();
+
+process.env.NODE_OPTIONS += ` --require ${'../../../.pnp.js'} --require ${require.resolve(`@yarnpkg/monorepo/scripts/setup-ts-execution`)}`;
```

Comment:
```suggestion
process.env.NODE_OPTIONS += ` --require ${require.resolve(`../../../.pnp.js`)} --require ${require.resolve(`@yarnpkg/monorepo/scripts/setup-ts-execution`)}`;
```

Reply:
That said, do we need to modify `NODE_OPTIONS`? Yarn shouldn't spawn subshells except those from the lifecycle scripts, which are configured to use the same Yarn as the one being executed anyway 🤔

Reply:
@arcanis I was unsure about this one myself. Lets remove settings `process.env.NODE_OPTIONS` for now, pushed the commit